source/NationalInstruments/jquery.flot.scattergraph.js

```diff
@@ -516,6 +517,40 @@ THE SOFTWARE.
 
                 ctx.restore();
             };
+
+            function findNearbyItems (plot, canvasX, canvasY, series, seriesIndex, radius, computeDistance, items) {
+                var ps = series[seriesIndex].datapoints.pointsize;
+                for (var j = 0; j < series[seriesIndex].data.length; j++) {
+                    for (var i = 0; i < dataLen; i++) {
+                        var x, y;
+                        if (isArrayOfObject) {
+                            x = series[seriesIndex].data[j][i].x;
+                            y = series[seriesIndex].data[j][i].y;
+                        } else if (isObjectOfArray) {
+                            x = series[seriesIndex].data[j].x[i];
+                            y = series[seriesIndex].data[j].y[i];
+                        } else if (isArrayOfArray) {
+                            x = series[seriesIndex].data[j][i][0];
+                            y = series.data[j][i][1];
```

Comment:
Should this be `series[seriesIndex]...`?

Reply:
Fixed